### ISSUE TITLE
Jiná práva v ČDK než ve zdrojovém Kramériovi

### ISSUE THREAD

**Issue by svetlym** [ceskaexpedice/ceska-digitalni-knihovna#34]
Dobrý den,

čtenářka nás upozornila, že dokument https://cdk.lib.cas.cz/view/uuid:6fafe190-a751-11dd-80bc-0030487be43a?page=uuid:82908df0-a765-11dd-98c8-0030487be43a je v ČDK uvedený jako neveřejný, přestože ve zdrojovém Kramériovi je veřejný, viz: https://digitalniknihovna.mlp.cz/view/uuid:6fafe190-a751-11dd-80bc-0030487be43a

Týká se to pravděpodobně i dalších dokumentů.

**Comment by zabak** [comment]
Tohle bude nutné řešit systémově. Práva se mění poměrně často, s nástupem DNNT to bude ještě divočejší (oběma směry). Budou nutné pravidelné aktualizace a partial indexování změněných práv.

**Comment by zabak** [comment]
Předpokládám, že toto bude řešeno až v ČDK7. Licence bude nutné synchronizovat na denní bázi. Předpokládám, že mimo "známých" licencí typu "public","dnnto" a "dnntt" budou vznikat různé lokální licence. ČDK je bude zachovávat a indexovat, ale nemusí je "znát". Určité riziko je v tom, že ve dvou krameriích budou licence označené stejně, takže ideální by bylo aby si je jednotlivé knihovny už u sebe označovaly svým prefixem.

**Comment by pavel-stastny** [comment]
@zabak Reseno ve dvou krocich:

1) Je domluva, ze na strane krameria budou globalni licence oznacovany bez prefixu, lokalni prefixem knihovny.  
2) V indexu vzniklo nove ridici pole **cdk.licences**, ktere drzi informace o licenci a knihovne, ktera danou licenci poskytla.  (Informaci drzi pro vsechny, lokalni i globalni)

Klient pri zobrazeni zamecku se ridi standardnim polem licenses - coz odpovida realite.  Pro rozhodnuti, kterou knihovnu uzivateli nabidnout pri pristupu na titul muze pouzit ridici pole cdk.licenses.

**Comment by zabak** [comment]
@pavel-stastny probíhá už nějaká automatická aktualizace změn u licencí?

**Comment by pavel-stastny** [comment]
@zabak  Ano, ale zdrojová knihovna musí mít nainstalované jádro, kde je zahrnuta oprava issue 
https://github.com/ceskaexpedice/kramerius/issues/952 resp. https://github.com/ceskaexpedice/kramerius/issues/954

**Comment by zabak** [comment]
@pavel-stastny Což je jaká verze jádra? Resp. které knihovny je potřeba požádat o aktualizaci?
